src/__tests__/components/fields/image-upload/image-upload.spec.tsx

```diff
@@ -52,7 +53,14 @@ const FrontendEngineWithEventListener = (props: ICustomFrontendEngineProps) => {
 		}
 	}, [eventListener, eventType]);
 
-	return <FrontendEngine {...otherProps} ref={formRef} />;
+	return (
+		<>
+			<FrontendEngine {...otherProps} data={data} ref={formRef} />
```

Comment:
tried to pass in the FrontendEngineWithCustomButton but need to change it to a forwardref component. Dint make the changes since it's only used here